src/amo/components/SearchForm/style.scss

```diff
@@ -0,0 +1,54 @@
+@import "~core/css/inc/mixins";
+@import "~amo/css/inc/vars";
+@import "~ui/css/vars";
+
+$item-padding: 5px 52px;
+
+.SearchForm-form {
+  background: $header-search-color;
+  border: 0;
+}
+
+.SearchInput-input,
+.SearchForm-form {
+  color: $base-color;
+  font-size: $font-size-m-smaller;
+  white-space: nowrap;
+}
+
+.SearchForm-suggestions {
+  &-list {
```

Comment:
Yes I was not 100% sure so I left things as is (_i.e._ stuck to my habits..). Thank you and sorry for reiterating this though.